webapp/src/components/Profile.tsx

```diff
@@ -25,20 +25,73 @@ import CompetencyRatings from './CompetencyRatings';
  * User data is currently hardcoded but it will be pulling data from the database (github_users)
  */
 
-const user = {
-  name: 'Matthew Morenez',
-  email: 'profile@example.com',
-  avatar:
-    'https://media.volinspire.com/images/95/e4/99/95e499b759ba57975a61c7bf66a3414dd5a2625e_profile.jpg',
-  github: 'https://github.com',
-  website: 'https://example.com',
-  summary:
-    'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Officia, sit impedit? Cupiditate veniam eaque suscipit eligendi. Sint delectus enim earum non repellendus nihil numquam libero odit temporibus et, natus eaque?',
-  linkedIn: 'https://linkedin.com',
-};
+//
+import { useState } from 'react';
+import useApiData from '../helpers/useApiData';
+import { EntityId, Profile as APIProfile } from '../types/api';
+import SessionContext from './SessionContext';
+
+// const user = {
+//   name: 'Matthew Morenez',
+//   email: 'profile@example.com',
+//   avatar:
+//     'https://media.volinspire.com/images/95/e4/99/95e499b759ba57975a61c7bf66a3414dd5a2625e_profile.jpg',
+//   github: 'https://github.com',
+//   website: 'https://example.com',
+//   summary:
+//     'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Officia, sit impedit? Cupiditate veniam eaque suscipit eligendi. Sint delectus enim earum non repellendus nihil numquam libero odit temporibus et, natus eaque?',
+//   linkedIn: 'https://linkedin.com',
+// };
 
 const Profile = () => {
-  const greeting = getGreeting(user.name);
+  const { principalId } = useContext(SessionContext);
```

Comment:
Is this getting the principal Id of the current user? This will be good for what we are trying to implement now, but maybe this should be passed as props into the Profile component at a later time